io.github.syntalos.yaml

```diff
@@ -0,0 +1,244 @@
+id: io.github.syntalos
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-23.08
+separate-locales: false
+
+command: syntalos
+rename-icon: syntalos
+rename-desktop-file: syntalos.desktop
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --filesystem=host
+  - --socket=system-bus
+  - --socket=session-bus
```

Comment:
These two needs exceptions https://docs.flathub.org/docs/for-app-authors/linter#finish-args-arbitrary-dbus-access

https://docs.flathub.org/docs/for-app-authors/linter#exceptions

Reply:
why are they needed?

Reply:
> These requirements aren't new though

You need the exception PR still I guess just rename the existing one https://github.com/flathub-infra/flatpak-builder-lint/blob/f52259b6f3c20fb1c40889d068633a94b893137b/flatpak_builder_lint/staticfiles/exceptions.json#L968-L969

But the ID needs to be fixed first, exceptions are tied to the ID.

Reply:
You can but I can't merge it until the ID is settled.

Reply:
Yeah, that's to be expected. Give me a day to talk to some people (I really want to avoid having to change that ID more than once).

Reply:
get them to buy a domain and use that....

Reply:
> get them to buy a domain and use that....

Well, that was exactly my plan, and it succeeded :-)
(it has been in the making before, and I just had to accelerate things a bit)